xmipp

```diff
@@ -418,15 +419,19 @@ def compileModule(Nproc,module):
             errorRegex = re.match(".*: fatal error: (.*): No such file or directory", l)
             if errorRegex:
                 missingPath = errorRegex.group(1)
-                print(red("\n'%s' file not found." % missingPath))
-                missingDir = missingPath.split(os.path.sep)[0]
-                missingFile = os.path.join(*missingPath.split(os.path.sep)[1:])
-                if missingDir in DEPENDENCIES.keys():
-                    depDir = os.path.join('src', missingDir)
-                    hint = ('cd %s ; git checkout -- %s ; cd -' % (depDir, missingFile)
-                            if os.path.isdir(depDir) else './xmipp get_dependencies')
-                    print(red("This file belongs to %s dependency. Please, "
-                              "try '%s' and re-compile." % (missingDir, hint)))
+                if lastError != missingPath:
```

Comment:
I would prefer here some comments what is expected contain and format of `missingPath`. It's kinda tricky to follow now